src/site/content/en/blog/richer-offline-w-periodic-background-sync/index.md

```diff
@@ -0,0 +1,458 @@
+  post will be updated as the implementation progresses.
+{% endAside %}
+
+
+Have you ever been in any of the following situations: riding a train or subway
+with flaky or no connectivity; being throttled by your carrier after watching
+too many videos; or living in a country where bandwidth struggles to keep up
+with the demand? If you have, then you've surely felt the frustration of getting
+certain things done on the web, and wondered why native apps so often do better
+in these scenarios.
+
+Native apps can fetch fresh content such as news articles or weather information
+ahead of time. Even if there's no network in the subway, you can still read the
+news.
+
+Periodic Background Sync enables web applications to periodically synchronize
+data in the background, bringing web apps closer to the behavior of a native
+app.
+
+## Current status
+
+The table below explains the current status of the Periodic Background Sync API.
+
+<table>
+<tr>
+<th markdown="block">
+Step
+</th>
+<th markdown="block">
+Status
+</th>
+</tr>
+<tr>
+<td markdown="block">
+1. Create explainer
+</td>
+<td markdown="block">
+<a href="https://github.com/beverloo/periodic-background-sync" >Complete</a>
+</td>
+</tr>
+<tr>
+<td markdown="block">
+2. Create initial draft of specification
+</td>
+<td markdown="block">
+In Progress
+</td>
+</tr>
+<tr>
+<td markdown="block">
+3. Gather feedback and iterate on design
+</td>
+<td markdown="block">
+In Progress
+</td>
+</tr>
+<tr>
+<td markdown="block">
+<strong>4. Origin trial</strong>
+</td>
+<td markdown="block">
+<strong><a href="https://developers.chrome.com/origintrials/#/view_trial/4048736065006075905">Started in Chrome 77<a></strong><br/>
+Expected to run through Chrome 80
+</td>
+</tr>
+<tr>
+<td markdown="block">
+5. Launch
+</td>
+<td markdown="block">
+Not started
+</td>
+</tr>
+</table>
+
+## Try it
+
+You can try periodic background sync with our [live demo
+app](https://webplatformapis.com/periodic_sync/periodicSync_improved.html).
+Before using it, make sure that:
+
+* You're using Chrome 77 or later.
+* You
+  [install](https://developers.google.com/web/fundamentals/app-install-banners/)
+  the web app before enabling periodic background sync. (The demo app's
+  author already took the step of signing up for the origin trial.)
+
+### Register for the origin trial {: #ot }
+
+{% include 'content/origin-trials.njk' %}
+
+{% include 'content/origin-trial-register.njk' %}
+
+## Concepts and usage
+
+Periodic background sync lets you show fresh content when a progessive web app
+or service worker-backed page is launched. It does this by downloading data in
+the background when the app or page is not being used. Without it, the app's
+content will refresh after launch, leading to content changing while the user is
+viewing it. Worse yet, the app may only show a content spinner.
+
+Without periodic background sync, web apps must use alternative methods to
```

Comment:
The previous sentence likewise starts with "without".

Reply:
It was easier to remove the one in the previous paragraph.